lua/astrocommunity/editing-support/nvim-regexplainer/nvim-regexplainer.lua

```diff
@@ -0,0 +1,9 @@
+return {
+  "bennypowers/nvim-regexplainer",
+  opts = {
```

Comment:
does the plugin start at all? 
you can use `ft =` to lazy load it on specific filetypes

Reply:
It starts when the key is pressed  because it's lazy loaded on `keys`

Reply:
imho it should run without activation